src/service.rs

```diff
@@ -47,6 +47,11 @@ where
     ///  - `name`=`<name>` a services's name
     #[serde(serialize_with = "crate::docker::serialize_as_json")]
     pub filters: HashMap<T, Vec<T>>,
+
+		/// Include service status, with count of running and desired tasks.
+    #[serde(serialize_with = "crate::docker::serialize_as_json")]
```

Comment:
I don't think this line is needed, it's used for serializing a `HashMap` type into the query parameter value that Docker expects, a boolean works as is.